uber/templates/registration/form.html

```diff
@@ -382,4 +385,28 @@
 
 </div>
 
+<script>
+var confirmAction = function(){
+    var check = confirm("Are you sure you'd like to check out this attendee?");
+    if (check){
+        $.ajax({
+            "url":"check_out_attendee_in_error",
+            "type":"POST",
+            "dataType":"json",
+            "data":{
+                "id":"{{ attendee.id }}",
+                "csrf_token":"{{ c.CSRF_TOKEN }}"
```

Comment:
Style note: there's a Javascript global variable named ``csrf_token`` which you can use in places like this.